debian/changelog

```diff
@@ -1,3 +1,10 @@
+regolith-session (1.1.12-4) trixie; urgency=medium
```

Comment:
We are dropping the support for `force-overwrite` on published repo, and as such this will cause an issue. What I think we should do is increment the last digit (i.e. `12 -> 13`) and updated the last sufix (e.g. `-4`) to be an indicator for different "live branches". For example:

- `x.y.z-1`: for `main`
- `x.y.z-2`: for `distro-foo`
- `x.y.z-3`: for `distro-bar`

Reply:
Ok, setting to `1.1.13-2`